test/scope.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*global describe,it*/
+
+const assert = require('insist');
+
+const Scope = require('../lib/scope');
+
+describe('Scope', function() {
+
+  describe('constructor', function() {
+
+    it('should accept a space-separated string', function() {
+      var s1 = Scope('a b c');
+      assert.deepEqual(s1.values(), ['a', 'b', 'c']);
+    });
+
+    it('should accept an array', function() {
+      var s1 = Scope(['a', 'b', 'c']);
+      assert.deepEqual(s1.values(), ['a', 'b', 'c']);
+    });
+
+    it('should accept a Scope instance', function() {
+      var s1 = Scope(['a', 'b', 'c']);
+      var s2 = Scope(s1);
+      assert.equal(s1, s2);
+    });
+
+  });
```

Comment:
Since the scope string comes from user input, I suggest adding some tests for weirdy values that might sneak through the Joi validator, like "foo         bar" and "profile::write:::".